spring-cloud-dataflow-server-core/src/main/resources/org/springframework/cloud/dataflow/server/db/migration/h2/V1__INITIAL_SETUP.sql

```diff
@@ -51,136 +51,7 @@ create table task_definitions (
-  VERSION BIGINT,
-  JOB_INSTANCE_ID BIGINT NOT NULL,
-  CREATE_TIME TIMESTAMP NOT NULL,
-  START_TIME TIMESTAMP DEFAULT NULL,
-  END_TIME TIMESTAMP DEFAULT NULL,
-  STATUS VARCHAR(10),
-  EXIT_CODE VARCHAR(2500),
-  EXIT_MESSAGE VARCHAR(2500),
-  LAST_UPDATED TIMESTAMP,
-  JOB_CONFIGURATION_LOCATION VARCHAR(2500) NULL,
-  constraint JOB_INST_EXEC_FK foreign key (JOB_INSTANCE_ID)
-  references BATCH_JOB_INSTANCE(JOB_INSTANCE_ID)
-);
-
-CREATE TABLE BATCH_JOB_EXECUTION_PARAMS (
-  JOB_EXECUTION_ID BIGINT NOT NULL,
-  TYPE_CD VARCHAR(6) NOT NULL,
-  KEY_NAME VARCHAR(100) NOT NULL,
-  STRING_VAL VARCHAR(250),
-  DATE_VAL TIMESTAMP DEFAULT NULL,
-  LONG_VAL BIGINT,
-  DOUBLE_VAL DOUBLE PRECISION ,
-  IDENTIFYING CHAR(1) NOT NULL ,
-  constraint JOB_EXEC_PARAMS_FK foreign key (JOB_EXECUTION_ID)
-  references BATCH_JOB_EXECUTION(JOB_EXECUTION_ID)
-);
-
-CREATE TABLE BATCH_STEP_EXECUTION (
-  STEP_EXECUTION_ID BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
-  VERSION BIGINT NOT NULL,
-  STEP_NAME VARCHAR(100) NOT NULL,
-  JOB_EXECUTION_ID BIGINT NOT NULL,
-  START_TIME TIMESTAMP NOT NULL,
-  END_TIME TIMESTAMP DEFAULT NULL,
-  STATUS VARCHAR(10),
-  COMMIT_COUNT BIGINT,
-  READ_COUNT BIGINT,
-  FILTER_COUNT BIGINT,
-  WRITE_COUNT BIGINT,
-  READ_SKIP_COUNT BIGINT,
-  WRITE_SKIP_COUNT BIGINT,
-  PROCESS_SKIP_COUNT BIGINT,
-  ROLLBACK_COUNT BIGINT,
-  EXIT_CODE VARCHAR(2500),
-  EXIT_MESSAGE VARCHAR(2500),
-  LAST_UPDATED TIMESTAMP,
-  constraint JOB_EXEC_STEP_FK foreign key (JOB_EXECUTION_ID)
-  references BATCH_JOB_EXECUTION(JOB_EXECUTION_ID)
-);
-
-CREATE TABLE BATCH_STEP_EXECUTION_CONTEXT (
-  STEP_EXECUTION_ID BIGINT NOT NULL PRIMARY KEY,
-  SHORT_CONTEXT VARCHAR(2500) NOT NULL,
-  SERIALIZED_CONTEXT LONGVARCHAR,
-  constraint STEP_EXEC_CTX_FK foreign key (STEP_EXECUTION_ID)
-  references BATCH_STEP_EXECUTION(STEP_EXECUTION_ID)
-);
-
-CREATE TABLE BATCH_JOB_EXECUTION_CONTEXT (
-  JOB_EXECUTION_ID BIGINT NOT NULL PRIMARY KEY,
-  SHORT_CONTEXT VARCHAR(2500) NOT NULL,
-  SERIALIZED_CONTEXT LONGVARCHAR,
-  constraint JOB_EXEC_CTX_FK foreign key (JOB_EXECUTION_ID)
-  references BATCH_JOB_EXECUTION(JOB_EXECUTION_ID)
-);
-
-CREATE TABLE task_execution_metadata (
-     ID BIGINT NOT NULL,
-     TASK_EXECUTION_ID BIGINT NOT NULL,
-     TASK_EXECUTION_MANIFEST CLOB,
-     PRIMARY KEY (ID),
-     CONSTRAINT TASK_METADATA_FK FOREIGN KEY (TASK_EXECUTION_ID)
-     REFERENCES TASK_EXECUTION(TASK_EXECUTION_ID)
-);
-
-CREATE SEQUENCE task_execution_metadata_seq START WITH 1 INCREMENT BY 1;
-CREATE SEQUENCE BATCH_STEP_EXECUTION_SEQ;
-CREATE SEQUENCE BATCH_JOB_EXECUTION_SEQ;
-CREATE SEQUENCE BATCH_JOB_SEQ;
-create index STEP_NAME_IDX on BATCH_STEP_EXECUTION (STEP_NAME);
-create index TASK_EXECUTION_ID_IDX on TASK_EXECUTION_PARAMS (TASK_EXECUTION_ID);
-
-CREATE TABLE BOOT3_TASK_EXECUTION  (
+CREATE TABLE TASK_EXECUTION  (
                                TASK_EXECUTION_ID BIGINT NOT NULL PRIMARY KEY ,
```

Comment:
Do you mind removing all of the formatting as it is hard to see what has actually changed. Mostly it is slight column type adjustments and a few new tables.

Reply:
I don't think there is file formatting changes.
2 significant changes here:
1. Remove Task V2 and Batch V4  schema declarations
2. Rename all Task V3 and Batch V5 schema declarations Removing the BOOT3_ prefixes.

Reply:
Looks good w/ the update.